src/nfa/operations.cc

```diff
@@ -179,10 +190,13 @@ bool Mata::Nfa::are_state_disjoint(const Nfa& lhs, const Nfa& rhs)
     return true;
 } // are_disjoint }}}
 
+//Should be in-place method? I would still name it complete, it is a verb too.
+//Alphabet should be optional, the automaton has an alphabet that would be used implicitly.
 bool Mata::Nfa::make_complete(Nfa& aut, const Alphabet& alphabet, State sink_state) {
     return Mata::Nfa::make_complete(aut, alphabet.get_alphabet_symbols(), sink_state);
 }
 
+//It is weird to have two finctions like this. Perhaps after we have alphabets finished, we could just pass an alphabet.
```

Comment:
Already discussed in the header file. Let us resolve it there.

Reply:
ok